<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SearchNumberScene</class>
 <widget class="QDialog" name="SearchNumberScene">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1951</width>
    <height>1300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>-140</y>
     <width>2291</width>
     <height>1361</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>../build-Final_Project-Desktop_Qt_5_12_12_MinGW_64_bit-Debug/Searching.PNG</pixmap>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton11">
   <property name="geometry">
    <rect>
     <x>1780</x>
     <y>40</y>
     <width>93</width>
     <height>91</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton12">
   <property name="geometry">
    <rect>
     <x>1340</x>
     <y>500</y>
     <width>191</width>
     <height>191</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton13">
   <property name="geometry">
    <rect>
     <x>60</x>
     <y>920</y>
     <width>121</width>
     <height>121</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit11">
   <property name="geometry">
    <rect>
     <x>390</x>
     <y>500</y>
     <width>951</width>
     <height>191</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <pointsize>14</pointsize>
    </font>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
